src/main/java/control/TimelineController.java

```diff
@@ -0,0 +1,102 @@
+package control;
+
+import data.Camera;
+import data.CameraShot;
+import data.CameraTimeline;
+import data.CameraType;
+import data.ScriptingProject;
+import gui.CameraShotBlock;
+import gui.ShotblockUpdatedEvent;
+import gui.TimelinesGridPane;
+import gui.RootPane;
+import gui.TimetableBlock;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Class that controls the timeline.
+ * @author alex
+ */
+public class TimelineController {
+
+    private TimelinesGridPane timelinePane;
+    private RootPane rootPane;
+
+    private List<CameraTimeline> cameraTimelines;
+
+    // Placeholder camera type until GUI allows personalized entry
+    private final CameraType defType = new CameraType("AW-HE130 HD PTZ", "It's an IP Camera", 0.5);
```

Comment:
Shouldn't this value be set in the contructor? This value can be `Timeline` dependant.